### PR TITLE
Timezone fixes

### PR DESCRIPTION
**Pull request by floppym**
A couple minor fixes for timezone detection in winpr. See commit messages for detail.

### CODE REVIEW

File: winpr/libwinpr/timezone/timezone.c [hunk 1]
```diff
@@ -323,7 +326,12 @@ static TIME_ZONE_ENTRY* winpr_detect_windows_time_zone(void)
 		char buf[1024] = { 0 };
 		const char* links[] = { buf };
 
-		snprintf(buf, ARRAYSIZE(buf), "%s%s", zipath, tzid);
+		if (tzid[0] == '/')
+			/* Full path given in TZ */
+			links[0] = tzid;
+		else
+			snprintf(buf, ARRAYSIZE(buf), "%s%s", zipath, tzid);
```
[the remark this answers is not in the document]
Comment (akallabeth) [reply winpr/libwinpr/timezone/timezone.c:333]: @floppym not an issue, we´ve added `clang-tidy` as a workflow to point outpossible  issues, but not all of the suggestions are really a problem.

File: winpr/libwinpr/timezone/timezone.c [hunk 2]
```diff
@@ -308,6 +308,9 @@ static TIME_ZONE_ENTRY* winpr_detect_windows_time_zone(void)
 			free(tzid);
 			tzid = NULL;
 		}
+		else if (tzid[0] == ':')
+			/* Remove leading colon, see tzset(3) */
```
Comment (akallabeth) [inline winpr/libwinpr/timezone/timezone.c:312]: better use braces if multiline, even if just a comment

File: winpr/libwinpr/timezone/timezone.c [hunk 3]
```diff
@@ -323,7 +326,12 @@ static TIME_ZONE_ENTRY* winpr_detect_windows_time_zone(void)
 		char buf[1024] = { 0 };
 		const char* links[] = { buf };
 
-		snprintf(buf, ARRAYSIZE(buf), "%s%s", zipath, tzid);
+		if (tzid[0] == '/')
+			/* Full path given in TZ */
```
Comment (akallabeth) [inline winpr/libwinpr/timezone/timezone.c:330]: better use braces if multiline, even if just a comment

File: winpr/libwinpr/timezone/timezone.c [hunk 4]
```diff
@@ -462,8 +470,8 @@ DWORD GetTimeZoneInformation(LPTIME_ZONE_INFORMATION lpTimeZoneInformation)
 	WLog_DBG(TAG, "tz not found, using computed bias %" PRId32 ".", tz->Bias);
 out_error:
 	free(dtz);
-	memcpy(tz->StandardName, L"Client Local Time", sizeof(tz->StandardName));
-	memcpy(tz->DaylightName, L"Client Local Time", sizeof(tz->DaylightName));
+	ConvertUtf8ToWChar("Client Local Time", tz->StandardName, ARRAYSIZE(tz->StandardName));
```
Comment (akallabeth) [inline winpr/libwinpr/timezone/timezone.c:473]: nice catch

**Review by akallabeth — APPROVED** [review]
overall LGTM, one nitpick ;)

File: winpr/libwinpr/timezone/timezone.c (hunk 2 above)
Comment (floppym) [reply winpr/libwinpr/timezone/timezone.c:313]: Added.

File: winpr/libwinpr/timezone/timezone.c (hunk 3 above)
Comment (floppym) [reply winpr/libwinpr/timezone/timezone.c:333]: Added.

**Review by akallabeth — APPROVED** [review]
LGTM, thank you!

**Pull request merged**